newm-server/src/main/kotlin/io/newm/server/features/user/model/UserIdBody.kt

```diff
@@ -1,11 +1,11 @@
 package io.newm.server.features.user.model
 
+import io.newm.server.typealiases.UserId
 import io.newm.shared.serialization.UUIDSerializer
 import kotlinx.serialization.Serializable
-import java.util.UUID
 
 @Serializable
 data class UserIdBody(
     @Serializable(with = UUIDSerializer::class)
-    val userId: UUID
+    val userId: UserId
 )
```

Comment:
Revert this one. This object comes back from a call from eveara and is technically a different userid than the ones in our users table.